lib/iris/_lazy_data.py

```diff
@@ -106,9 +172,12 @@ def as_lazy_data(data, chunks=None, asarray=False):
 
```

Comment:
Can you update the doc string for this function. The "chunk" explanation" is out of date